src/index.d.ts

```diff
@@ -70,9 +70,18 @@ declare namespace connect {
    * @param scope The instance object to be set as the scope of the function.
    * @param method The method to be encapsulated.
    */
-  function hitch<T extends Function>(scope: object, method: T): T;
+  function hitch<T extends (...args: any[]) => any>(scope: object, method: T): T;
 
   interface Core {
+    /**
+     * Indicates whether `amazon-connect-streams` communicated successfully with the CCP.
+     * 
+     * **Important:** This does not mean that the CCP has been fully initialized, and it's mostly used internally by `amazon-connect-streams`.
+     * It's not recommended that you use this field to detect whether the CCP has been initialized.
+     * As an alternative, consider using `connect.agent()`.
+     */
+    readonly initialized: boolean;
```

Comment:
Same as above. There are ways to get around this in TypeScript (e.g. explicitly using `// @ts-ignore`) when referencing `initialized` if the client choose to use this undocumented/internal property.

As your comment suggested there are better alternatives and the customer should use the documented methods.